mtenn/config.py

```diff
@@ -404,192 +404,27 @@ def _check_grouped(values):
 
 class GATModelConfig(ModelConfigBase):
     """
-    Class for constructing a graph attention ML model. Note that there are two methods
-    for defining the size of the model:
-
-    * If single values are passed for all parameters, the value of ``num_layers`` will
-      be used as the size of the model, and each layer will have the parameters given
-
-    * If a list of values is passed for any parameters, all parameters must be lists of
-      the same size, or single values. For parameters that are single values, that same
-      value will be used for each layer. For parameters that are lists, those lists will
-      be used
-
-    Parameters passed as strings are assumed to be comma-separated lists, and will first
-    be cast to lists of the appropriate type, and then processed as described above.
-
-    If lists of multiple different (non-1) sizes are found, an error will be raised.
-
-    Default values here are the default values given in DGL-LifeSci.
+    Class for constructing a GAT ML model. Default values here are based on the values
+    in DGL-LifeSci.
```

Comment:
DGL-LifeSci gone now?

Reply:
the defaults are still based on the defaults in that package, even though we're not using their code anymore